### DISCUSSION TITLE
cannot execute binary file: Exec format error

### DISCUSSION THREAD

**Discussion by cosmos-star** [pawelsalawa/sqlitestudio#4490] [Q&A] [answered]
### Details
After install I execute sqlitestudio
steven@DEVELOPMENT-JETSON:/opt/SQLiteStudio/$ ./sqlitestudio 
bash: ./sqlitestudio: cannot execute binary file: Exec format error

Any ideas why this is happening? It seemed a rather straight forward install.

root@DEVELOPMENT-JETSON:/opt# wget https://github.com/pawelsalawa/sqlitestudio/releases/download/3.3.3/sqlitestudio-3.3.3.tar.xz
root@DEVELOPMENT-JETSON:/opt# tar -xf sqlitestudio-3.3.3.tar.xz
opt/SQLiteStudio directort was created and all files loaded

steven@DEVELOPMENT-JETSON:/opt/SQLiteStudio$ ls -al
total 428
drwxr-xr-x 10 1001 docker   4096 Apr 11  2021 .
drwxr-xr-x  6 root root     4096 Aug 11 10:49 ..
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 iconengines
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 imageformats
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 lib
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 platforms
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 platformthemes
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 plugins
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 printsupport
-rwxr-xr-x  1 1001 docker  76000 Apr 11  2021 sqlitestudio
-rwxr-xr-x  1 1001 docker 317664 Apr 11  2021 sqlitestudiocli
drwxr-xr-x  2 1001 docker   4096 Apr 11  2021 styles

### Operating system
Ubuntu 18.04 ARM64 Jetson Nano

### SQLiteStudio version
sqlitestudio-3.3.3.tar.xz

**Comment by pawelsalawa** [comment, marked as answer]
You're running ARM64 system. Binary distribution for Linux is compiled for x64. If you want ARM64, you need to compile it by ourself.
